platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/MerkleRoot.java

```diff
@@ -57,6 +57,14 @@ public interface MerkleRoot extends MerkleInternal {
     @NonNull
     PlatformStateModifier getWritablePlatformState();
 
+    /**
+     * Gets the Roster state that is writable, to be called by users who need to modify the Roster state.
+     *
+     * @return the writable Roster state.
+     */
+    @NonNull
+    RosterStateModifier getWritableRosterState();
```

Comment:
After our earlier chat offline, I found that since the components that will use the RosterState has not been built yet, it's better to remove this method for now. Removed.